Updated data asheet link and foot print for LTS-6960HR

Misca1234 opened the pull request:
Added the data sheet link for LTS-6980HR
Re directed the foot print to the same as LTS-6980HR

LTS-6960HR
https://datasheet.octopart.com/LTS-6960HR-Lite-On-datasheet-11803242.pdf



------------
Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [ ] Provide a URL to a datasheet for the symbol(s) you are contributing
- [ ] An example screenshot image is very helpful
- [ ] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required

evanshultz commented:
A few questions about `Display_Character.lib`:
1. Why does line 1560 contain a text string that is just spaces?
1. `LTS-6960HR` needs the footprint filter updated.
1. Three symbols have 40mil pin name offset. Would you mind changing them to 20mil?

Misca1234 commented:
Fixed, but I had to change the text distance on all, they eother had 30 or 40 mil
Now, this might might cascade a lot of additional issues with those I changed

evanshultz commented:
Surprisingly no issues according to Travis. Thanks!
